File: smelly/assertion-roulette/CSharpXUnit/CSharpXUnit.Tests/Tests/PollsChoiceModelImpl_ESTest.cs
[Collection("EvoRunner")]
public class PollsChoiceModelImpl_ESTest : PollsChoiceModelImpl_ESTest_scaffolding
{
    [Fact(Timeout = 4000)]
    public void test00()
    {
        var pollsChoiceImpl0 = new PollsChoiceImpl();
        var pollsChoiceWrapper0 = new PollsChoiceWrapper(pollsChoiceImpl0);
        pollsChoiceWrapper0.SetChoiceId(1L);
        var boolean0 = pollsChoiceImpl0.Equals(pollsChoiceWrapper0);
        Assert.Equal(1L, pollsChoiceImpl0.GetChoiceId());
        Assert.True(boolean0);
    }
}

File: refactored/assertion-roulette/python-pytest/PollsChoiceModelImpl_ESTest.py
import pytest

class TestPollsChoiceModelImpl:
    @pytest.mark.timeout(4000)
    def test_00(self):
        polls_choice_impl_0 = PollsChoiceImpl()
        polls_choice_wrapper_0 = PollsChoiceWrapper(polls_choice_impl_0)
        polls_choice_wrapper_0.set_choice_id(1)
        boolean_0 = polls_choice_impl_0.equals(polls_choice_wrapper_0)
        assert polls_choice_impl_0.get_choice_id() == 1, "Explanation message"
        assert boolean_0, "Explanation message"
